pjp.tm10/src/main/webapp/Styles&StyleSheets/css/All.css

```diff
@@ -0,0 +1,27 @@
+@charset "ISO-8859-1";
+/* Question 1 */
+
+#q1div
+{
+	text-align: Center;
+	Font-Size : 27px;
+	Font-family: arial;
+}
+
+/* Question2 */
+
+.q2div
+{
+	text-align: Center;
+	font-size: 25px;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5662373)